Source/dnd/dndGameModeBase.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/GameModeBase.h"
+#include "dndGameModeBase.generated.h"
+
+UCLASS()
+class DND_API AdndGameModeBase : public AGameModeBase
```

Comment:
'AdndGameModeBase"

Should it be as is or:

'ADndGameModeBase' or 'ADnDGameModeBase'?